tests/fixture-app/app.controller.ts

```diff
@@ -11,7 +13,8 @@ export class AppController {
 
   @Get(':id')
   @OtelMethodCounter()
-  example() {
+  example(@OtelCounter('example_counter') counter: Counter) {
```

Comment:
Howdy! Would you mind adding the example with a custom description here so that it's actually going through the fixed paths?

Reply:
Hi, done ! I also added an assertion for the description in the metrics output